src/Main.elm

```diff
@@ -342,7 +353,10 @@ switchTopRow model colors theme =
         , height <| fillPortion 1
         ]
         [ Element.column [ Font.center ]
-            [ Element.row [ spacingXY 10 0 ]
+            [ Element.row
+                [ spacingXY 10 0
+                , height <| px 100
```

Comment:
Not too happy with this here, but it seems needed to position the button correctly near the top of the containing element.